Makefile

```
@@ -0,0 +1,9 @@
+BITNAMI_REDIS_RELEASE=18.1.6
+
+gen-redis-manifest:
+	docker run -ti --rm \
+		--volume $${PWD}/redis:/opt/manifests \
+		--env BITNAMI_REDIS_RELEASE=${BITNAMI_REDIS_RELEASE} \
+		--workdir=/opt/manifests \
+		--entrypoint=/bin/sh \
+		alpine/helm ./gen-yaml/gen.sh
```

Nit/unrelated/can do later if you want: if you add execute permissions on `./gen-yaml/gen.sh` you can simplify(? not sure if it's any less clear) the invocation

```suggestion
		--entrypoint=./gen-yaml/gen.sh \
		alpine/helm
```

---

Good shout! [Applied](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/14/commits/cdbefb7ffca7f2b270ceccd4482db53435030e0c#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R8)